README.md

```diff
@@ -370,31 +370,31 @@ Integration flow:
+1. Initialize the payment form with the Credit Card payment method ID and return URL (necessary for 3-D Secure transactions).
+   - The return URL redirects the user once payment is completed on the 3-D Secure’s side.
+   - The `headlessCheckout.form.init` method returns the form object that can be used for future work with the payment form.
 1. Subscribe to events of the `NextActions` form to receive notifications about the next payment flow steps.
-   - Next action with the `show_fields` type informs means that form needs to render extra fields, e.g. for Brazilian credit cards. Partner must remove previously added fields and render new fields for this step.
-   - Next action with the `redirect` type means that form will be redirected to complete payment according **3DS 1.0** secure procedure. The correct return url must be provided to return from 3DS verification flow when it completed.
-   - Next action with the `3DS` type means user card must be checked according **3DS 2.0** procedure. Partner will manage opening 3DS window by himself.
+   - The Next action with the `show_fields` type means that the form needs to render extra fields, e.g., for Brazilian credit cards. A partner must remove previously added fields and render new fields for this step.
+   - The Next action with the `redirect` type means the form is redirected to complete payment according to the **3DS 1.0** secure procedure. The correct return URL must be provided to return from the 3-D Secure verification flow when it is completed.
+   - The Next action with the `3DS` type means the user card must be checked according to the **3DS 2.0** procedure. The partner is responsible for opening a 3-D Secure window.
 1. Add the form fields component to the HTML markup.
    - Use the form object that was returned by `headlessCheckout.form.init` method to get form fields.
    - Use fields with the `isMandatory` flag to get required fields only.
-   - Use the `<psdk-text>` component to render form fields if required. For field named `card_number` you must use `<psdk-card-number>` component.
+   - Use the `<psdk-text>` component to render form fields if required. For a field named `card_number` you must use the `<psdk-card-number>` component.
 1. Add the `<psdk-submit-button>` form submit button to the HTML markup.
 1. Add the `<psdk-status>` component to the HTML markup to see the payment status.
 1. Create a `return` page.
-1. Add to it `<psdk-finance-details>`, `<psdk-status>` and `<psdk-legal>` components to show payment status.
-1. Initialize `headlessCheckout.init` to init library and after that set accessToken at `headlessCheckout.setToken`
+1. Add the `<psdk-finance-details>`, `<psdk-status>` and `<psdk-legal>` components to the created `return` page to show a payment status.
+1. Initialize `headlessCheckout.init` to initialization library and set accessToken at `headlessCheckout.setToken`.
```

Comment:
sorry, but i think it will be better to write like: "Set accessToken at `headlessCheckout.setToken`. Run `headlessCheckout.init` to initialize headless checkout library"